db/db_block_cache_test.cc

```diff
@@ -1845,10 +1845,7 @@ TEST_P(DBBlockCachePinningTest, TwoLevelDB) {
   const int kNumKeysPerFile = kBlockSize * kNumBlocksPerFile / kKeySize;
 
   Options options = CurrentOptions();
-  // `kNoCompression` makes the unit test more portable. But it relies on the
-  // current behavior of persisting/accessing dictionary even when there's no
-  // (de)compression happening, which seems fairly likely to change over time.
-  options.compression = kNoCompression;
+  options.compression = kLZ4Compression;
```

Comment:
This test needs to be conditionally compiled now. Example: https://github.com/facebook/rocksdb/blob/fc40165614e3aecc1475d61f895f508b28247c0d/utilities/blob_db/blob_db_test.cc#L634-L725